### PR TITLE
Wagtail 6.0

### PR DESCRIPTION
**Pull request by katdom13**
[Support Ticket](https://torchbox.monday.com/boards/1124794299/pulses/1396935095)

### Summary of changes

- Drop versions of Wagtail < 6.0
- Add Wagtail 6.0 support
- Add Django 5.0 support

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -2,9 +2,9 @@
 min_version =  4.11
 
 env_list =
-    python{3.8,3.9,3.10}-dj{32}-wagtail{41,52}
-    python{3.11}-dj{41,42}-wagtail{52,60}
-    python{3.12}-dj{42,50}-wagtail{52,60}
+    python{3.8,3.9,3.10,3.11,3.12}-django3.2-wagtail5.2
```
Comment (zerolab) [inline tox.ini:5]: considering Django 3.2 is soon EOL, I don't think we should test with the full suite of python versions

```suggestion
    python{3.8,3.9,3.10}-django3.2-wagtail5.2
```

File: tox.ini [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+    python{3.8,3.9,3.10,3.11}-django4.2-wagtail{5.2,6.0}
+    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.0}
```
Comment (zerolab) [inline tox.ini:7]:
```suggestion
    python{3.11,3.12}-django5.0-wagtail{5.2,6.0}
```

**Review by zerolab — CHANGES_REQUESTED** [review]
Thank you for this. A couple of minor changes

File: tox.ini (hunk 2 above)
Comment (katdom13) [reply tox.ini]: Hi @zerolab ,
Thanks for the suggestions. Applied those here:
https://github.com/torchbox/wagtail-headless-preview/pull/62/commits/4af7d7bf65694b7ce603e39c81cc746d8cfdd4e3

**Review by zerolab — APPROVED** [review]

**Pull request merged**